tg_bot/modules/locks.py

```diff
@@ -94,6 +98,77 @@ def locktypes(bot: Bot, update: Update):
     update.effective_message.reply_text("\n - ".join(["Locks: "] + list(LOCK_TYPES) + list(RESTRICTION_TYPES)))
 
 
+@run_async
+@user_admin
+@loggable
+def add_whitelist(bot: Bot, update: Update):
+    chat = update.effective_chat  # type: Optional[Chat]
+    user = update.effective_user  # type: Optional[User]
+    message = update.effective_message  # type: Optional[Message]
+    entities = message.parse_entities([MessageEntity.URL])
+    added = []
+    for url in entities.values():
+        if sql.add_whitelist(chat.id, url):
+            added.append(url)
+    if added:
+        message.reply_text("Added to whitelist:\n- "+'\n- '.join(added))
+        return "<b>{}:</b>" \
+               "\n#WHITELIST" \
+               "\n<b>Admin:</b> {}" \
+               "\nWhitelisted:\n<pre>- {}</pre>".format(html.escape(chat.title),
+                                                         mention_html(user.id, user.first_name),
+                                                         html.escape('\n- '.join(added)))
+    else:
```

Comment:
unnecessary else; youve returned in the if statement. the following code can be un-indented by one block.